controllers/idm_controller_errors.go

```diff
@@ -0,0 +1,17 @@
+package controllers
```

Comment:
TODO This file is not used; Review if it will be used or necessary, else remove this.